test/video.test.js

```diff
@@ -103,6 +105,17 @@ describe('Video', () => {
 			video.videoEl.className.should.equal('class-one class-two o-video__video');
 		});
 
+		it('should support the playsinline option', () => {
```

Comment:
Could we add two more tests here?
A test that checks it doesn't add the attributes if `playsinline` is `false`.
A test that checks `playsinline` is set to `false` by default (I.E. if not passed in as an option).